### PR TITLE
Fix issue where stats db created in wrong location

### PR DESCRIPTION
**Pull request by max-hoffman**
fixes: https://github.com/dolthub/dolt/issues/8324

### CODE REVIEW

**Comment by coffeegoddd** [comment]
@max-hoffman __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>36b5e01</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>36b5e01</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

**Comment by coffeegoddd** [comment]
@max-hoffman __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>957d78e</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>957d78e</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

File: integration-tests/bats/stats.bats [hunk 1]
```diff
@@ -116,6 +116,52 @@ teardown() {
...
+    cd repo2
+
+    dolt sql -q "insert into xy values (0,0), (1,1)"
+
+    dolt sql -q "call dolt_stats_restart()"
+
+    run dolt sql -r csv -q "select count(*) from dolt_statistics"
+    [ "$status" -eq 0 ]
+
+    run stat .dolt/repo2
```
Comment (zachmu) [inline integration-tests/bats/stats.bats:129]: Test is a little wonky. Probably should look for the file you want, not the one you don't.

**Review by zachmu — APPROVED** [review]
LGTM

File: integration-tests/bats/stats.bats (hunk 1 above)
Comment (max-hoffman) [reply integration-tests/bats/stats.bats:129]: yeah, unfortunately we make the files on `dolt init`. i added a second test that verifies that a round-trip picks up the expected data

**Comment by coffeegoddd** [comment]
@max-hoffman __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>81b4c81</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>81b4c81</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

**Pull request merged**